introduction.md

```diff
@@ -1,3 +1,7 @@
 Hello! I am here to learn about GitHub.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?